### ISSUE TITLE
As a Connor O'Day, I want the ability to one-click set a bounty as "hypercharge mode" and the product will do the work of promoting it to the community.

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#7135]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a Connor O'Day, I want the ability to one-click set a bounty as "hypercharge mode" and the product will do the work of promoting it to the community.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
Connors time is limited, and we need to give him the abiliy to quickly promote bounties he cares about.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*:
New

### Current Behavior
[comment]: # (Describe what actually happened.)
No hypercharge mode

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Admin flag on bounties where someone can set a bool "hypercharge mode" and save
When a bounty is in hypercharge mode (ie. it is not done/cancelled), it will be
- tweeted 2x/day automatically
- gitcoinbot will write a pinned post about it
- there will be secret/daily offers for the bounty
- The bounty will be featured in the daily email.
- (what else, anywhere else we should promote it?)

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
see above

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)
how much quicker / more consistently do hypercharge bounties get picked up?

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

**Comment by connoroday** [comment]
Looks good. One thought is there are often multiple bounties that need attention, so how can this work with say 3 bounties? Maybe they can alternate day by day, or maybe I can set a time range for how long and when each will be promoted?

Other ways to promote:
- Notify users who have worked with that org before
- Auto feature on hack explorer/normal prize explorer

**Comment by owocki** [comment]
thanks @connoroday

**Comment by owocki** [comment]
@zoek1 any interest in this one

**Comment by zoek1** [comment]
Of course, happy to build something that helps Connor and also this looks pretty interesting! @owocki

**Comment by owocki** [comment]
@zoek1 pls do. can we aim for a PR by sunday?  feel free to PM me how much you think it'll cost, im pretty easy as long as it's reasonable

**Comment by zoek1** [comment]
Almost done in https://github.com/gitcoinco/web/pull/7150
@connoroday can you provide me the texts to create the offers and also to create a pinned post? After that, i'll record a demo and will be done this feature

**Comment by owocki** [comment]
@zoek1 Offers

```
Title: "Checkout this bounty worth AMOUNT TOKEN_NAME"
Description: Work on "BOUNTY_TITLE" and receive AMOUNT TOKEN_NAME

BOUNTY_DESCRIPTION[0:...500 chars]
```

Pinned Post from gitcoinbot
```
Work on "BOUNTY_TITLE" and receive AMOUNT TOKEN_NAME

BOUNTY_DESCRIPTION[0:...500 chars]
```
